src/material/bottom-sheet/testing/bottom-sheet-harness-filters.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {BaseHarnessFilters} from '@angular/cdk/testing';
+
+export interface BottomSheetHarnessFilters extends BaseHarnessFilters {}
```

Comment:
For a follow-up: it would be nice to add a filter based on the aria-label (if only so that there's _some_ filter for this). Similar for dialog.

Reply:
I didn't add any filters, because only one bottom sheet can be open at a time.

Reply:
That's a good point. Would it make sense to just remove the `with` then, since it would never make sense to need to filter?

Reply:
I'd rather leave the `with` for consistency with our other harnesses. Besides, even if only one can be open at a time, it can still be useful to ensure that you're grabbing the harness for the one you expect